src/System.IO.Pipes/src/System/IO/Pipes/AnonymousPipeServerStream.Windows.cs

```diff
@@ -15,14 +13,31 @@ namespace System.IO.Pipes
     /// </summary>
     public sealed partial class AnonymousPipeServerStream : PipeStream
     {
+        // bufferSize is used as a suggestion; specify 0 to let OS decide
```

Comment:
Make these XML comments?

Reply:
Also, should the '0' letting the OS decide be called out on the new API?

Reply:
Actually those comments are wrong. Removed. Not sure why we had them before.